ash/src/extensions/khr/video_encode_queue.rs

```diff
@@ -0,0 +1,84 @@
+//! <https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/VK_KHR_video_encode_queue.html>
+
+use core::mem;
+use core::ptr;
+
+use crate::prelude::*;
+use crate::vk;
+use crate::RawMutPtr;
+
+impl crate::khr::video_encode_queue::Device {
+    /// <https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/vkCmdEncodeVideoKHR.html>
+    #[inline]
+    #[doc(alias = "vkCmdEncodeVideoKHR")]
+    pub unsafe fn cmd_encode_video(
+        &self,
+        command_buffer: vk::CommandBuffer,
+        encode_info: &vk::VideoEncodeInfoKHR<'_>,
+    ) {
+        (self.fp.cmd_encode_video_khr)(command_buffer, encode_info)
+    }
+
+    /// Retrieves the length of the byte slice to pass to [`get_encoded_video_session_parameters`][Self::get_encoded_video_session_parameters].
+    pub unsafe fn get_encoded_video_session_parameters_len(
```

Comment:
Looking at other implementations, we may have coincidentally forgotten `#[inline]` on some `_len()` functions?

Reply:
Presumably wherever I copy/pasted from :)